src/Parcel/Parcel.spec.js

```diff
@@ -106,19 +107,55 @@ describe('Parcel', function() {
   describe('.findWithLastActiveMortgageByBorrower', function() {
     beforeEach(() => new ParcelService().insertMatrix(0, 0, 10, 10))
 
-    it('should return parcels with mortgages open/claimed by borrower', async function() {
```

Comment:
I saw in other spec files we are using arrow functions, this file is using es5 ones, is that for any particular reason? If not I think is better to be consistent across all the other code.

Reply:
Oh! yep, we are using both, we may create an issue or Trello card to make the change in another PR SQL.spec has arrow functions too